server/core/Redis.ts

```diff
@@ -11,11 +11,23 @@ const REDIS_CONNECTION_URL = process.env.REDIS_CONNECTION_URL as string;
 const REDIS_TIMEOUT_RECONNECTION = parseInt(process.env.REDIS_TIMEOUT_RECONNECTION as string);
 
 export default class RedisWorks {
-    private _client: RedisClient;
-    private readonly _redisStore: RedisStoreType;
```

Comment:
we remove readonly because we don't initialize it in constructor anymore?

Reply:
yes, we should use "readonly" modifier only for constant fields which will be assigned in constructor (if we don`t assign the variable in contructor - we must assign it anywhere in the class)